Cargo.toml

```diff
@@ -42,6 +42,7 @@ hw_tests = []
 dangerous_hw_tests = ["hw_tests"]
 sev = []
 snp = []
+openssl = ["dep:openssl"]
```

Comment:
Would we like to mention in the commit message that openssl is being added as a feature in this PR?

Reply:
Wasn't it already a feature? What does this line do?

Reply:
@tylerfanelli this makes it an explicit feature defined for the library.